tests/security/apparmor/aa_easyprof.pm

```diff
@@ -0,0 +1,84 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: AppArmor profile generation made easy testing
+# Maintainer: Wes <whdu@suse.com>
+# Tags: poo#36895, tc#1621144
+
+use strict;
+use base "apparmortest";
+use testapi;
+use utils;
+use version_utils qw(is_sle is_leap);
+
+sub run {
+    my ($self) = @_;
+
+    my $aa_easyprof_jsonopt;
+    my $aa_easyprof_out;
+    my $aa_easyprof_tmp_json = "/tmp/manifest.json";
+    my $aa_easyprof_result   = "/tmp/output";
+    my @aa_easyprof_cmd;
+    my $aa_easyprof_cmd_convert;
+
+    # For new apparmor version, easyprof support convert from a manifest
+    # json file
+    if (is_sle('<15') or is_leap('<15.0')) {    # apparmor < 2.11.95
```

Comment:
Looks like behavior of your tests depending on **package** version not OS . So maybe you should just ask zypper what apparmor version installed ? 
Current logic will fail when/if SLE 12 SP4/SP5 will get latest apparmor.
BUT maybe you want test to fail in that case ? 
Just thinking loud you may ignore this comment .

Reply:
Actually it works for SLE12 SP4, because SLE12 SP4 is still with old AppArmor version. But I agree with check package version is the best practice. I will think how to improve it in future.